.github/workflows/test.yml

```diff
@@ -9,1670 +9,177 @@ on:
-
-      - name: Install and configure
-        run: |
-          source ci/ciLibrary.source
-          install_configure
-
-      - name: Prepare for coverage reporting
-        if: ${{ env.ENABLE_COVERAGE == 'true' }}
-        run: |
-          source ci/ciLibrary.source
-          configure_coverage
-
-      - name: Unit testing
-        run: |
-          source ci/ciLibrary.source
-          build_test unit
-        if: ${{ success() || failure() }}
-
-      - name: E2e testing
-        run: |
-          source ci/ciLibrary.source
-          build_test e2e
-        if: ${{ success() || failure() }}
-
-      - name: Api testing
-        run: |
-          source ci/ciLibrary.source
-          build_test api
-        if: ${{ success() || failure() }}
-
-      - name: Fixtures testing
-        run: |
-          source ci/ciLibrary.source
-          build_test fixtures
-        if: ${{ success() || failure() }}
-
-      - name: Services testing
-        run: |
-          source ci/ciLibrary.source
-          build_test services
-        if: ${{ success() || failure() }}
-
-      - name: Validators testing
-        run: |
-          source ci/ciLibrary.source
-          build_test validators
-        if: ${{ success() || failure() }}
-
-      - name: Controllers testing
-        run: |
-          source ci/ciLibrary.source
-          build_test controllers
-        if: ${{ success() || failure() }}
-
-      - name: Common testing
-        run: |
-          source ci/ciLibrary.source
-          build_test common
-        if: ${{ success() || failure() }}
+    - uses: actions/checkout@v4
+    - name: Install PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.configurations.php }}
+    - name: Report PHP Version
+      run: php -v
+    - name: Install npm package
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+    - name: Main build
+      run: |
+        . ci/ciLibrary.source
+        composer_github_auth
+        main_build
+    - name: CCDA build
+      run: |
+        . ci/ciLibrary.source
+        ccda_build
+    - name: Dockers environment start
+      run: |
+        . ci/ciLibrary.source
+        dockers_env_start
+    - name: Wait for MySQL to initialize
+      if: ${{ matrix.configurations.database == 'mysql' }}
+      run: |
+        echo "Waiting 60 seconds for MySQL to initialize..."
+        sleep 60
+    - name: Install and configure
+      run: |
+        . ci/ciLibrary.source
+        install_configure
+    - name: Prepare for coverage reporting
+      if: ${{ env.ENABLE_COVERAGE == 'true' }}
+      run: |
+        . ci/ciLibrary.source
+        configure_coverage
+    - name: Unit testing
+      if: ${{ success() || failure() }}
+      run: |
+        . ci/ciLibrary.source
+        build_test unit
+    - name: E2e testing
+      if: ${{ (success() || failure()) && ! contains(fromJson(vars.SKIP_E2E_CONFIGS || '[]'), matrix.configurations.docker_dir) }}
```

Comment:
@adunsulag @bradymiller If I did this right, then if you create a repo variable named `SKIP_E2E_CONFIGS` and its value is a valid json string that contains the docker dir names of the configs for which you want to disable e2e, then those tests should be skipped. 
![image](https://github.com/user-attachments/assets/16546b48-6801-4838-8210-0882be234a17)